LICENSE.txt

```diff
@@ -48,6 +48,8 @@ licenses (all MPL 2.0 compatible):
 
     vendor/promise.js       : MIT
 
+    vendor/resize-observer-polyfill : MIT
+
```

Comment:
You should probably also include the path in the section below "The following license texts are included".